PointOfDispensingApp/src/components/CreateLocationResourcePage.vue

```diff
@@ -0,0 +1,301 @@
+          <v-col cols="8">
+            <v-text-field outlined dense 
+              v-model="search_postalCode"
+            ></v-text-field>
+          </v-col>
+        </v-row>
+      </v-col>
+      <v-col cols="1">
+        <!--blank column for spacing-->
+      </v-col>
+      <v-col cols="5">
+        <v-row no-gutters>
+          <v-col cols="4">
+            <div class="secondary--text">State</div>
+          </v-col>
+          <v-col cols="8">
+            <v-select
+              :items="stateOptions"
+              outlined
+              dense
+              v-model="search_state"
+            ></v-select>
+          </v-col>
+          <v-col cols="4">
+            <div class="secondary--text">Organization</div>
+          </v-col>
+          <v-col cols="8">
+            <v-text-field outlined dense 
+              v-model="search_organization"
+            ></v-text-field>
+          </v-col>
+        </v-row>
+      </v-col>
+    </v-row> 	
+    <v-row no-gutters>
+      <v-col cols="12">
+        <v-btn color="accent" @click="retrieveLocationResources()">
+          Search
+        </v-btn>
+      </v-col>
+    </v-row>
+    <v-row>
+      <v-col cols="12">
+        <v-divider></v-divider>
+      </v-col>
+    </v-row>
+    <v-row>
+      <v-col cols="12">
+        <template>
+          <v-data-table
+            @click:row="rowClick"
+            item-key="siteID"
+            single-select
+            :items="locationResourceLookupTable"
+            :headers="headers"
+            class="elevation-1"
+            :footer-props="{
+              'items-per-page-options':[5]
+            }"
+            :items-per-page="5"
+          ></v-data-table>
+        </template>
+      </v-col>
+    </v-row>
+    <v-row>
+      <v-col cols="12">
+        <v-dialog
+          v-model="createLocationResourceDialog"
+          persistent
+          max-width="70%"
+          >
+            <template v-slot:activator="{ on, attrs }">
+              <v-btn color="accent" v-bind="attrs" v-on="on">
+                Create location resource
+              </v-btn>
+            </template>
+            <v-card>
+              <v-card-title class="headline grey lighten-2 justify-center">
+                Create Location Resource
+              </v-card-title>
+              <v-card-text>
+                <v-container>
+                  <v-row no-gutters>
+                    <v-col cols="2">
+                      <div class="secondary--text">Site ID</div>
+                    </v-col>
+                    <v-col cols="9">
+                      <v-text-field outlined dense 
+                        v-model="identifier_value"
+                        placeholder="(Unique identifier)"
+                      ></v-text-field>
+                    </v-col>
+                  </v-row>
+                  <v-row no-gutters>
+                    <v-col cols="2">
+                      <div class="secondary--text">Organization</div>
```

Comment:
Why are we asking for organization? Isn't this assumed to be part of the group that set it up?

Reply:
This is where we would specify the organization as "Waukesha County", for example.  As part of this administrator configuration, I think we need to accept this as an input.  Otherwise, we would somehow/somewhere need to hardcode this in for each prospective organization we work with.

Reply:
I'm in the camp of removing it since I see that as an implementation process. I can see that as being slightly advantageous if they wanted a personalized name for their site for display purposes. I don't think we'd use it as an identifier though. 

Let's for now keep this as is and we can refine later on.